include/cell.tcc

```diff
@@ -648,9 +661,9 @@ inline Eigen::Matrix<double, Tdim, 1>
   // Check if the first trial xi is just outside the box
   for (unsigned i = 0; i < nr_xi.size(); ++i) {
     if (nr_xi(i) < -1. && nr_xi(i) > -1.001)
-      nr_xi(i) = -0.999999999999;
```

Comment:
Can we not use machine epsilon here? Maybe keep it at 1E-12? Would that be OK? Because this is not checking whether the particle is at the edge, but just making sure that there is a good starting point for the NR iterations. So an arbitrary number that makes sense should be fine.

Reply:
Done